src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -473,9 +475,9 @@ AthensSurfaceExamples class >> example8 [
 			can pathTransform loadIdentity.
 			can setPaint: paint.
 
-			can drawShape: (0@0 corner: 200@200).
-		
-			Display getCanvas drawImage: surf asForm  at: 0@0 ]
+			can drawShape: (0@0 corner: 200@200)].
+
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?